### ISSUE TITLE
Question/feature request: skip tailing separators when key is empty

### ISSUE THREAD

**Issue by sochtends** [korv/Jabref-plugins#8]
Hi,
I use the following name pattern
[authors]\_[year]\_[journal]\_[volume]\_[title]
However, when [volume] is missing, there is still the remaining separator between [volume] and [title] in the file name. Is there a way to skip it when the key is empty?

**Comment by korv** [comment]
I don't think so. I suppose you will have to change the pattern for the volumeless entries.
